packages/neuron-wallet/src/services/tx-wallet.ts

```diff
@@ -0,0 +1,460 @@
+import { TransactionWithoutHash, WitnessArgs, OutPoint, Cell, Input, DepType } from 'types/cell-types'
+import WalletService, { Wallet } from 'services/wallets'
+import WalletsService from 'services/wallets'
+import { IsRequired } from 'exceptions'
+import NodeService from './node'
+import ConvertTo from 'types/convert-to'
+import { serializeWitnessArgs } from '@nervosnetwork/ckb-sdk-utils'
+import { TransactionPersistor, TransactionsService, TransactionGenerator } from './tx'
+import NetworksService from './networks'
+import { AddressPrefix } from 'models/keys/address'
+import LockUtils from 'models/lock-utils'
+import AddressService from './addresses'
+import { Address } from 'database/address/address-dao'
+import { PathAndPrivateKey } from 'models/keys/key'
+import AddressesService from 'services/addresses'
+import { CellIsNotYetLive, TransactionIsNotCommittedYet } from 'exceptions/dao'
+import TypeConvert from 'types/type-convert'
+import FeeMode from 'models/fee-mode'
+import DaoUtils from 'models/dao-utils'
+import TransactionSize from 'models/transaction-size'
+import TransactionFee from 'models/transaction-fee'
+import logger from 'utils/logger'
+import Keychain from 'models/keys/keychain'
+
+export default class TxWalletService {
```

Comment:
`TxWallet` is confusing. Please consider a better name such as `TransactionService`.

Reply:
A class already named `TransactionService`, how about `TransactionSender` ? It's a class combine transaction and wallet (to sign transactions).

Reply:
`TransactionSender` looks good to me 👍

Reply:
updated